integration/test-cleanup.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
+apps=$( auth0 apps list --format json --no-input)
+
+for app in $( echo "${apps}" | jq -r '.[] | @base64' ); do
+    _jq() {
+     echo "${app}" | base64 --decode | jq -r "${1}"
+    }
+
+    clientid=$(_jq '.ClientID')
+    name=$(_jq '.Name')
+
+    if [[ $name = integration-test-* ]]
```

Comment:
I'm wondering if this would only remove the apps created during the same tests execution, otherwise it might cause issues with other CI running at the same time. 
thoughts?

Reply:
@jfatta if we have multiple integration tests on this tenant running at once, we'll want something smarter that deletes by client id. When we get CI running we'll also want to split out the test and cleanup steps so that the cleanup step is sure to run even if the tests fail.